packages/paste-core/components/chat-composer/stories/ConversationsUIKit/index.stories.tsx

```diff
@@ -132,11 +132,10 @@ export const ConversationsUIKitExample: StoryFn = () => {
       <MinimizableDialog aria-label="Live chat">
         <MinimizableDialogHeader>Live chat</MinimizableDialogHeader>
         <MinimizableDialogContent>
-          <Box overflowY="scroll" maxHeight="size50" tabIndex={0}>
+          <Box overflow="hidden" overflowY="scroll" maxHeight="size50" tabIndex={0}>
```

Comment:
Stops the weird scroll behaviour as you animate in new bubbles

Reply:
But does it scroll?